tests/basic/basic_test.go

```diff
@@ -572,6 +574,70 @@ var _ = Describe("{CordonStorageNodesDeployDestroy}", func() {
 	})
 })
 
+var _ = Describe("{SecretsVaultFunctional}", func() {
+	var testrailID, runID int
+	var contexts []*scheduler.Context
+
+	const (
+		secretsProvider       = "vault"
+		portworxContainerName = "portworx"
+	)
+
+	BeforeEach(func() {
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+		isOpBased, _ := Inst().V.IsOperatorBasedInstall()
+		if !isOpBased {
+			k8sApps := apps.Instance()
+			deployments, err := k8sApps.ListDaemonSets("kube-system", metav1.ListOptions{
```

Comment:
nit: we are listing daemonSets not deployments, the variable name should match that

Reply:
updated.